.github/workflows/schedule-netlify-build.yml

```diff
@@ -0,0 +1,15 @@
+# .github/workflows/schedule-netlify-build.yml
+
+name: Schedule Netlify Build
+on:
+  schedule:
+    # Customize schedule expression at crontab.guru
+    # This one runs at 800 UTC daily
+    - cron: "0 8 * * *"
+jobs:
+  build:
+    name: Call Netlify build hook
+    runs-on: ubuntu-latest
+    steps:
+      - name: Curl request
+        run: curl -X POST -d {} https://api.netlify.com/build_hooks/616df66f7945d53c55e1e5b2
```

Comment:
@chrisaddams I've added this as a webhook on publish over on acast. Does this make this schedule redundant?

<img width="704" alt="image" src="https://user-images.githubusercontent.com/9591167/137884988-dac96d2f-bdf3-488e-b06e-37eaa849d2b9.png">